app/src/main/res/layout/layout_current_weather.xml

```diff
@@ -112,20 +104,15 @@
         android:layout_height="wrap_content"
         android:textSize="@dimen/text18"
         android:textStyle="bold"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="1"
-        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toStartOf="@+id/guideEnd"
         app:layout_constraintTop_toBottomOf="@+id/tvHumidity"
         tools:text="53" />
 
-    <ImageView
-        android:id="@+id/ivCurrentWeatherCondition"
-        android:layout_width="@dimen/dimen200"
-        android:layout_height="@dimen/dimen200"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintVertical_bias="0.5"
-        tools:ignore="ContentDescription" />
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/guideEnd"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:layout_constraintGuide_end="@dimen/dimen16" />
```

Comment:
Nice. Instead of giving marginEnd to all children I added it in the main view. But with guidelines no need for it.